src/NodeJSService.ts

```diff
@@ -203,6 +213,19 @@ FAUSTWP_SECRET_KEY=${secretKey}
 
 			// Next.js needs to be restarted after writing the env file.
 			await siteProcessManager.restart(this._site);
+
+			// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+			// @ts-ignore
+			if (this._site.customOptions.atlasUrl) {
```

Comment:
Will you add some comments here and in other place to explain why you check for `atlasUrl`?